### ISSUE TITLE
API returns empty result on more than one status in filter

### ISSUE THREAD

**Issue by pheunitb** [rapidmail/rapidmail-apiv3-client-php#1]
$queryResult returns a non empty query result (OK)
```
            $queryResult = $recipientService->query([
                'recipientlist_id' => $listId,
                'status' => ['new']
            ]);
```

$queryResult also returns a non empty query result (OK)
```
            $queryResult = $recipientService->query([
                'recipientlist_id' => $listId,
                'status' => ['active']
            ]);
```

In this case $queryResult always returns an empty query result (BUG)
```
            $queryResult = $recipientService->query([
                'recipientlist_id' => $listId,
                'status' => ['new','active']
            ]);
```
And also this case $queryResult always returns an empty query result (BUG)
```
            $queryResult = $recipientService->query([
                'recipientlist_id' => $listId,
                'status' => 'new,active'
            ]);
```

It would also be very helpful to implement 'all' as vaid status as it is used on the rapidmail website. Example: https://my.rapidmail.de/recipient/list.html?recipientlist_id=123&status=all

**Comment by dendemann** [comment]
Thanks for your report @pheunitb . I forwarded your request to our support team and they will look into this issue.

**Comment by eth8505** [comment]
@pheunitb I just published version 1.0.1 of the apiclient that should resolve the status filter issue.

We will not be implementing the "all" filter, we only use that in our frontend code for technical reasons, and it just doesn't feel right to have something that says "no filter" if you can just leave the filter out alltogether.
